src/pages/TradeV3/perps/DepositWithdraw.tsx

```diff
@@ -517,6 +427,31 @@ const CloseTradingAccount: FC<{ setDepositWithdrawModal: Dispatch<SetStateAction
           tw="w-[240px] h-8.75 mt-2 sm:mt-3"
         >
           Close Account
+        </Button> */}
+        {/* className={cn('min-w-[170px] w-full h-[30px]')}
+                  variant="default"
+                  colorScheme="blue"
+                  size="lg"
+                  onClick={() => handlePlaceOrder()}
+                  disabled={buttonState !== ButtonState.CanPlaceOrder} */}
+        <Button
+          loading={isLoading}
+          disabled={
+            !checkboxChecked ||
+            isLoading ||
+            !fundsWithdrawn ||
+            !clearedAllOpenPositions ||
+            !openOrdersCleared ||
+            !traderAddresses[0]
+          }
+          className={cn('min-w-[140px] w-full h-[30px]')}
```

Comment:
conflicting size/tw/classname properties. Remove and consolidate classes of tw, under classname and choose the height that is required